src/modules/inventing/QuickInvent.svelte

```diff
@@ -0,0 +1,79 @@
+<script>
+  import daDoInvent from '../_dataAccess/daDoInvent';
+  import { sendEvent } from '../support/fshGa';
+  import { slide } from 'svelte/transition';
+
+  export let max;
+  export let recipeID;
+  let amountToInvent = 1;
+  let successes = 0;
+  let failures = 0;
+  let progress = 0;
+  let errorMessage = '';
+
+  function reset() {
+    successes = 0;
+    failures = 0;
+    progress = 0;
+    errorMessage = '';
+  }
+
+  function maxInvent() {
+    amountToInvent = max;
+    sendEvent('inventing', 'maxInventButton');
+  }
+
+  async function quickInvent() {
+    sendEvent('inventing', 'quickInvent');
+    if (!amountToInvent) { return; }
+    reset();
+    const requests = Array(amountToInvent).fill(recipeID);
+    requests.reduce(async (prev, recipe) => {
+      const prevData = await prev;
+      if (!prevData || prevData.s === true) {
+        const data = await daDoInvent(recipe);
+        if (data.s === false) errorMessage = data.e.message;
+        else if (data.r.success_count > 0) successes += 1;
+        else failures += 1;
+        progress = 100 * ((successes + failures) / amountToInvent);
+        return data;
+      }
+      return prevData;
+    }, Promise.resolve());
+  }
+</script>
+<form class="fshCenter" on:submit|preventDefault={quickInvent} style="margin-top: 12px;">
```

Comment:
seems odd to use a global style in a svelte component.
why use a global style and an inline style on the same element?
you can just use a style block in svelte.
i do not love the inline styles. is there a reason for this?